static/app/gettingStartedDocs/javascript/react.mdx

```diff
@@ -0,0 +1,169 @@
+import {PRODUCT, ProductSelection} from 'sentry/components/onboarding/productSelection';
+import {DocumentationSection} from 'sentry/components/onboarding/documentationSection';
+import {useState} from 'react';
+
+<ProductSelection
+  defaultSelectedProducts={[PRODUCT.PERFORMANCE_MONITORING, PRODUCT.SESSION_REPLAY]}
+/>
+
+## Install
+
+Sentry captures data by using an SDK within your application’s runtime.
+
+```bash
+# Using yarn
+yarn add @sentry/react
+
+# Using npm
+npm install --save @sentry/react
+```
+
+## Configure
+
+Initialize Sentry as early as possible in your application's lifecycle.
+
+<DocumentationSection activeProductSelection={props.activeProductSelection}>
+
+```javascript
+import { createRoot } React from "react-dom/client";
+import React from "react";
+import * as Sentry from "@sentry/react";
+import App from "./App";
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+});
+
+const container = document.getElementById(“app”);
+const root = createRoot(container);
+root.render(<App />)
+```
+
+</DocumentationSection>
+
+<DocumentationSection products={[PRODUCT.SESSION_REPLAY, PRODUCT.PERFORMANCE_MONITORING]} activeProductSelection={props.activeProductSelection}>
+
+```javascript
+import { createRoot } React from "react-dom/client";
+import React from "react";
+import * as Sentry from "@sentry/react";
+import App from "./App";
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  integrations: [
+    new Sentry.BrowserTracing({
+      // Set `tracePropagationTargets` to control for which URLs distributed tracing should be enabled
+      tracePropagationTargets: ["localhost", /^https:\/\/yourserver\.io\/api/],
+    }),
+    new Sentry.Replay(),
+  ],
+  // Performance Monitoring
+  tracesSampleRate: 1.0, // Capture 100% of the transactions, reduce in production!
+  // Session Replay
+  replaysSessionSampleRate: 0.1, // This sets the sample rate at 10%. You may want to change it to 100% while in development and then sample at a lower rate in production.
+  replaysOnErrorSampleRate: 1.0, // If you're not already sampling the entire session, change the sample rate to 100% when sampling sessions where errors occur.
+});
+
+const container = document.getElementById(“app”);
+const root = createRoot(container);
+root.render(<App />)
+```
+
+</DocumentationSection>
```

Comment:
@stephanie-anderson In this concept, if SDK engineers have to update a document, they would have to work with DocumentSection and its content.

Here is an example where we have the code snippet for configuring Sentry with Error Monitoring, Session Replay and Performance Monitoring 🙂 Error Monitoring is not opt-out.